mlir/lib/Dialect/Mesh/IR/MeshOps.cpp

```diff
@@ -7,6 +7,8 @@
 //===----------------------------------------------------------------------===//
 
 #include "mlir/Dialect/Mesh/IR/MeshOps.h"
+#include "mlir/Dialect/Mesh/IR/MeshDialect.h"
```

Comment:
Nit: this should be in the next group below, "mlir/Dialect/Mesh/IR/MeshOps.h" should be alone at the top.

Reply:
Thank you. Moved it.